engine/redis/engine.go

```diff
@@ -306,3 +300,20 @@ func (e *Engine) DumpInfo(out io.Writer) error {
 	enc.SetIndent("", "    ")
 	return enc.Encode(metadata)
 }
+
+func (e *Engine) writeJob2Storage(job engine.Job) error {
+	now := time.Now().Unix()
```

Comment:
Need to add the pool name into storage as well, or all jobs will fan out to all engines.